### PR TITLE
Nft redemptions modal

### PR DESCRIPTION
**Pull request by tomquirk**
### to test

1. enable feature flag `jb.enableFeatureFlag('nftRedeem')`
2. buy an nft here http://localhost:3000/v2/p/146
3. Go to new NFT section and click "redeem for ETH"
4. Try to redeem
5. Check if you get your money back

### CODE REVIEW

File: src/components/v2v3/V2V3Project/ManageNftsSection/RedeemNftsModal/RedeemNftCard.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      const url = openIpfsUrl(cidFromIpfsUri(tokenUri))
+      const response = await axios.get(url)
+      const tierMetadata: IPFSNftRewardTier = response.data
+
+      return tierMetadata
+    },
+  )
+}
+
+export function RedeemNftCard({
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/ManageNftsSection/RedeemNftsModal/RedeemNftCard.tsx:28]: slight tech debt, a lot of duplicaiton from the normal NFT card. o well, needed for now

File: src/components/v2v3/V2V3Project/ManageNftsSection/ManageNftsSection.tsx [hunk 2]
```diff
@@ -49,9 +58,27 @@ export function ManageNftsSection() {
...
             </div>
+            {nftRedeemAllowed ? (
+              <div>
+                <Button
+                  size="small"
+                  onClick={() => {
+                    setRedeemNftsModalVisible(true)
+                  }}
+                >
+                  Redeem for ETH
```
Comment (johnnyd-eth) [inline src/components/v2v3/V2V3Project/ManageNftsSection/ManageNftsSection.tsx:69]: Trans?

File: src/components/v2v3/V2V3Project/ManageNftsSection/RedeemNftsModal/RedeemNftCard.tsx (hunk 1 above)
Comment (johnnyd-eth) [reply src/components/v2v3/V2V3Project/ManageNftsSection/RedeemNftsModal/RedeemNftCard.tsx:28]: Too easy

**Review by johnnyd-eth — APPROVED** [review]
Tested and all working well! LFG

**Pull request merged**